### PR TITLE
[WICKET-7101] modify schema definition

### PR DESCRIPTION
**Pull request by reiern70**

### CODE REVIEW

File: wicket-core/src/main/resources/META-INF/wicket.xsd [hunk 1]
```diff
@@ -102,6 +103,15 @@
 		</simpleType>
 	</attribute>
 
+	<attribute name="auto" type="boolean">
+		<annotation>
+			<documentation>Can be used in HTML <![CDATA[<label]]> to indicate the label will be automatically updated
```
Comment (martin-g) [inline wicket-core/src/main/resources/META-INF/wicket.xsd:108]:
```suggestion
			<documentation>The attribute can be used in HTML &lt;label&gt; to indicate the label will be automatically updated
```

Comment (reiern70) [reply wicket-core/src/main/resources/META-INF/wicket.xsd:108]: Hum... other elements use CDATA



Maybe change all of them too?

Comment (reiern70) [reply wicket-core/src/main/resources/META-INF/wicket.xsd:108]: Screenshot from 2024-03-28 15-48-36

Comment (martin-g) [reply wicket-core/src/main/resources/META-INF/wicket.xsd:108]: I updated the suggestion

**Pull request merged**